cmd/internal/cli/pull.go

```diff
@@ -48,8 +49,21 @@ var (
 	unauthenticatedPull bool
 	// PullDir is the path that the containers will be pulled to, if set
 	PullDir string
+	// PullArch is the architecture for which containers will be pulled from the
+	// SCS library
+	PullArch string
```

Comment:
this is a bad pattern that has crept into this particular part of the code: these variables don't need to be exported, we are passing them down to the appropriate functions as arguments.

Reply:
Do you want me to change them all to private then / or set this one just private? I was being consistent here.